frontend/public/components/chargeback.tsx

```diff
@@ -317,23 +316,30 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
 
   render() {
     const {obj} = this.props;
-    const phase = _.get(obj, ['status', 'phase']);
-
+    const conditions = _.get(obj, ['status', 'conditions']);
+    const isFinished = _.some(conditions, {reason: 'Finished'});
+    const errorCondition = _.find(conditions, (condition) => {
+      return condition.reason === 'GenerateReportFailed' ||
+        condition.reason === 'InvalidReport' ||
+        condition.reason === 'ReportingPeriodUnmetDependencies' ||
+        condition.type === 'Failure';
```

Comment:
Can't we check for value true instead of specific reason strings?

Reply:
no, I don't think so.  Not sure which field returns true except `status`, but that always seems to return `False`:
```
status:
  conditions:
    - lastTransitionTime: '2019-05-06T17:28:04Z'
      lastUpdateTime: '2019-05-06T17:28:04Z'
      message: >-
        spec.reportingEnd (2019-12-30 23:59:59 +0000 UTC) must be after
        spec.reportingStart (2020-01-01 00:00:00 +0000 UTC)
      reason: InvalidReport
      status: 'False'
      type: Running
```

Reply:
We can't check `condition.type` without also checking `status`